appserver/payara-appserver-modules/microprofile/opentracing/src/main/java/fish/payara/microprofile/opentracing/jaxrs/client/JaxrsClientBuilderDecorator.java

```diff
@@ -165,7 +164,7 @@ public Client build() {
 
     private boolean requestTracingPresent() {
         try {
-            ServiceLocator locator = PAYARA_SERVICES.getBasicServiceLocator();
+            ServiceLocator locator = Globals.getStaticBaseServiceLocator();
```

Comment:
This will start HK2 base service locator initialization. Don't use it. I hate Globals API and sooner or later I will delete that class, because I must always open it and look on it's content :-D
Better use new PayaraTracingServices().isTracingAvailable(). That should be safe (if Globals.getDefaultBaseServiceLocator() OR same getDefaultHabitat() returns null -> all services are null -> false)